pkg/kubelet/server/stats/handler.go

```diff
@@ -215,13 +215,17 @@ func (h *handler) handleStats(request *restful.Request, response *restful.Respon
 // If "only_cpu_and_memory" GET param is true then only cpu and memory is returned in response.
 func (h *handler) handleSummary(request *restful.Request, response *restful.Response) {
 	onlyCPUAndMemory := false
-	request.Request.ParseForm()
+	var err error
+	err = request.Request.ParseForm()
```

Comment:
Thoughts on `err := request.Request.ParseForm()`? I think that's more concise/idiomatic, with no readability sacrifice.

Reply:
Ok with this :)